packages/cody/src/lib/hooks/command-files/shared/commands/__service__/__fileName__.ts__tmpl__

```diff
@@ -22,4 +23,5 @@ export const <%= serviceNames.className %><%= className %>RuntimeInfo: CommandRu
   desc: <%= className %>Desc,
   factory: <%= propertyName %>,
   schema: <%= className %>Schema,
+  uiSchema: <%= className %>UiSchema,
```

Comment:
Cody now adds a uiSchema from prooph board command metadata to the command runtime info of the app. The uiSchema is used to configure json-schema-form